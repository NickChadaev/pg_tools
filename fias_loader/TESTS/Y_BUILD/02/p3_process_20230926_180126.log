2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_02 postgres /home/n.chadaev@abrr.local/Y_BUILD/02/stage_gar_c_02.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Башкортостан Респ, id_region: 4, Execute: /home/n.chadaev@abrr.local/Y_BUILD/02/stage_gar_c_02.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_REESTR_OBJECTS_20230925_e57120fb-3042-4ca4-898c-ff583d3008fe.XML
[18:50:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_ADDR_OBJ_20230925_bd5c7a3c-b862-4153-841c-f41a0fe6bed3.XML
[18:50:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_HOUSES_20230925_152bbf04-bc61-4293-a72b-8b01ee8900d1.XML
[19:28:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_STEADS_20230925_801a1929-8046-442a-a7db-72fde54bf8c8.XML
[19:45:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_APARTMENTS_20230925_ebcf2edc-09c3-4616-af47-d5bdaa170063.XML
[20:02:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_ROOMS_20230925_28aa94b8-512f-403c-8e60-30a4d91757c2.XML
[20:03:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_CARPLACES_20230925_6f733242-53c8-45c0-8d96-aedac6300cc9.XML
[20:03:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_ADM_HIERARCHY_20230925_f5538a79-53e7-49e6-aca8-6da12cb81cfa.XML
[20:57:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_MUN_HIERARCHY_20230925_0c3d3ca5-9fb7-4b68-b91b-afa4789637c7.XML
[21:47:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_CHANGE_HISTORY_20230925_1fc1b0be-f457-4dba-b965-06c648b54c96.XML
[23:57:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_ADDR_OBJ_DIVISION_20230925_4c160dd9-3cd3-447c-8929-bd43643c1e33.XML
[23:57:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_ADDR_OBJ_PARAMS_20230925_bedd1ec2-441d-4703-ac51-e274d0809a59.XML
[00:05:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_STEADS_PARAMS_20230925_41446106-b5c4-488a-8995-8f49df84d0a7.XML
[03:08:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_HOUSES_PARAMS_20230925_9dcc7a85-0a40-45b2-b249-6f9ccb5f326b.XML
[06:10:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_APARTMENTS_PARAMS_20230925_65590560-814f-4ac6-b74e-c86394e592ce.XML
[07:06:15] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_ROOMS_PARAMS_20230925_0a23b6d8-63dc-4661-ade9-10a516e9e2c9.XML
[07:07:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_CARPLACES_PARAMS_20230925_1ccaea57-d97e-4448-8d74-7da3031f7bf9.XML
[07:07:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/02/AS_NORMATIVE_DOCS_20230925_19ea4c12-4ee4-41cf-9598-d14e9a7fec01.XML
[07:15:58] ...  -- Data finish
[07:15:58] ...  -- Set LOGGED
[07:18:20] ... Stop process
--------------------------------------------------------
