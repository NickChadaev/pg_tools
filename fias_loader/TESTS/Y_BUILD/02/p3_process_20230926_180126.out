 save_gar_version 
------------------
               71
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                71
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 4003821
gar_fias.as_addr_obj: 60042
gar_fias.as_houses: 2848034
gar_fias.as_steads: 1371492
gar_fias.as_apartments: 1412656
gar_fias.as_rooms: 34742
gar_fias.as_carplaces: 10297
gar_fias.as_adm_hierarchy: 4159722
gar_fias.as_mun_hierarchy: 4090431
gar_fias.as_change_history: 10817827
gar_fias.as_addr_obj_division: 223
gar_fias.as_addr_obj_params: 695722
gar_fias.as_steads_params: 15559559
gar_fias.as_houses_params: 15391200
gar_fias.as_apartments_params: 4653356
gar_fias.as_rooms_params: 68976
gar_fias.as_carplaces_params: 7773
gar_fias.as_normative_docs: 715334
